KC_Liberation_Master_Framework/karmakut/deployable_field_hospitals/server/init.sqf

```diff
@@ -0,0 +1,61 @@
+karma_deployableFieldHospitals_server_fieldHospitalsByPlayerUID = createHashMap;
+
+karma_deployableFieldHospitals_server_deployFieldHospital = {
+    params [
+        ["_player", objNull]
+    ];
+
+    if (isNull _player) exitWith {};
+
+    private _playerUID = getPlayerUID _player;
+    if (!isNull (karma_deployableFieldHospitals_server_fieldHospitalsByPlayerUID getOrDefault [_playerUID, objNull])) exitWith {
```

Comment:
Should store the resulting field hospital in a variable and also return it back to the client in case they deployed a field hospital and disconnected and rejoined, to rehydrate their state.